docs/Way-of-Working/UbiquitousLanguage.md

```diff
@@ -0,0 +1,227 @@
+and used for statistical analysis or data modeling. The data of a dataset can be gathered from many sources and is
+typically stored in some kind of data structure, most commonly a table, where the columns correspond to different
+variables, and the rows to different data entries.
+
+### Deep Learning (DL)
+
+Deep learning is a subset of machine learning algorithms based on
+multilayered [artificial neural networks (ANN)](https://www.datacamp.com/blog/data-science-glossary#artificial-neural-networks-(ann))
+that are largely inspired by the structure of the brain. ANN are very flexible and can learn from huge amounts of data,
+to deliver highly accurate outputs. They are often behind some data science and machine learning use cases such as image
+or sound recognition, language translation, and other advanced problems.
+
+### Feature
+
+A feature is an independent variable used as an input in a machine learning model. For example, if we’re predicting the
+likelihood of diabetes using height, weight, and sugar intake—height, weight, and sugar intake are features.
+
+### Inference
+
+Inference is the operation by which you generate text with a generative LLM. The input is a trained model and a sequence
+of tokens, and the output is the next predicted token. If you want to generate a whole sequence with inference, you just
+generate one token at a time.
+
+### Linear Regression
+
+Linear regression is a regression algorithm that deals with modeling a linear relationship between a continuous target
+variable and one or several continuous features. A typical example of data science using linear regression is price
+prediction based on various input attributes.
+
+### Logistic Regression
+
+Logistic regression is a regression algorithm that uses a logistic function on the input features to predict the class
+probability or directly the class label for the target variable. In the second case, the output represents a set of
+categories instead of continuous values, meaning that the logistic regression acts here as a classification technique. A
+typical data science use case for logistic regression is predicting the likelihood of customer churn.
+
+### Normalization
+
+Normalization is the process of rescaling the data so that all the attributes have the same scale. Normalization is
+necessary for making a meaningful comparison between the attributes and also is required for some machine learning
+algorithms.
+
+### Overfitting
+
+Overfitting refers to when a model learns too much information from the training set including potential noise and
+outliers. As a result, it becomes too complex, too conditioned on the particular training set, and fails to adequately
+perform on unseen data. Overfitting leads to high variance on
+the [bias-variance tradeoff](https://www.datacamp.com/blog/data-science-glossary#bias-variance-tradeoff).
+
+### Precision
```

Comment:
If you want to mention precision maybe also mention related metrics such as recall and F1 scores